### PR TITLE
Improving tutorial text

### PR DESCRIPTION
**Pull request by camilamaia**
# Description

Improving tutorial text. Applying @maiakovsky 's suggestions

Preview: https://deploy-preview-81--gracious-cray-a1ce6a.netlify.app/tutorials/page1.html

### CODE REVIEW

File: _tutorials/page10.md [hunk 1]
```diff
@@ -104,14 +104,14 @@ Also, this should be the folder structure:
...
-  - .env
-  - scanapi-report.html
-  - scanapi.conf
-  - scanapi.yaml
-  - snippets.yaml
+|── .env
+|── scanapi-report.html
+|── scanapi.conf
+|── scanapi.yaml
+|___  snippets.yaml
```
Comment (maiakovsky) [inline _tutorials/page10.md:111]: Small typo, the last one, `snippets.yaml`, is indented here

File: _tutorials/page10.md [hunk 2]
```diff
@@ -104,14 +104,14 @@ Also, this should be the folder structure:
...
 ```
 
-Worth notice that you can recursively include files. In our example, the `snippets.yaml` could be composed of as many includes you want.
+Worth noticing that you can recursively include files. In our example, the `snippets.yaml` could be composed of as many includes you want.
```
Comment (maiakovsky) [inline _tutorials/page10.md:114]: maybe: as many includes `as` you want.

File: _tutorials/page12.md [hunk 3]
```diff
@@ -47,12 +47,12 @@ The folder structure should look like this now:
...
-  - scanapi-report.html
-  - scanapi.conf
-  - scanapi.yaml
-  - snippets.yaml
+|── .env
+|── csv_template.jinja
+|── scanapi-report.html
+|── scanapi.conf
+|── scanapi.yaml
+|___  snippets.yaml
```
Comment (maiakovsky) [inline _tutorials/page12.md:55]: again the last item is indented. that is not expected, right?

File: _tutorials/page12.md [hunk 4]
```diff
@@ -75,13 +75,13 @@ This should be the final folder structure:
...
-  - scanapi.conf
-  - scanapi.yaml
-  - snippets.yaml
+|── .env
+|── csv_template.jinja
+|── scanapi-report.csv
+|── scanapi-report.html
+|── scanapi.conf
+|── scanapi.yaml
+|___  snippets.yaml
```
Comment (maiakovsky) [inline _tutorials/page12.md:84]: Same here

File: _tutorials/page6.md [hunk 5]
```diff
@@ -26,10 +26,10 @@ The folder structure should look like this now:
...
 ```
 - scanapi (root directory)
-  - .env
-  - scanapi-report.html
-  - scanapi.conf
-  - scanapi.yaml
+|── .env
+|── scanapi-report.html
+|── scanapi.conf
+|___  scanapi.yaml
```
Comment (maiakovsky) [inline _tutorials/page6.md:32]: again :)

File: _tutorials/page5.md [hunk 6]
```diff
@@ -53,9 +54,9 @@ The folder structure should look like this now:
 
 ```
 - scanapi (root directory)
-  - .env
-  - scanapi-report.html
-  - scanapi.yaml
+|── .env
+|── scanapi-report.html
+|___  scanapi.yaml
```
Comment (maiakovsky) [inline _tutorials/page5.md:59]: here as well

File: _tutorials/page10.md (hunk 1 above)
Comment (camilamaia) [reply _tutorials/page10.md:111]: @maiakovsky I did it on purpose, to make it aligned in the rendered version

image

I tried to add directly the tree output, but look how it gets:

image

**Pull request merged**